backend/geneaprove/models/__init__.py

```diff
@@ -54,7 +54,7 @@ class Project (GeneaProveModel):
                                          through="Researcher_Project")
     name = models.CharField(max_length=100)
     description = models.TextField(null=True)
-    scheme = models.ForeignKey(Surety_Scheme, default=1)
+    scheme = models.ForeignKey(Surety_Scheme, default=1, on_delete=models.PROTECT)
```

Comment:
Since it used to default to CASCADE with older versions, perhaps we should keep CASCADE at least for now ?

Reply:
CASCADE always makes me irrationally nervous. :} But yes, that's probably the best course. I just wanted to hear your opinion before going with it. Changed.